04_numeronym/kasunfdo/main_test.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote(`[a11y K8s abc]
```

Comment:
I think when there is a single `\n` terminator on the string, this would read better as `"[a11y K8s abc]\n"`, rather than have the embedded newline in a raw string. Raw strings can be easier to read when there are multiple lines, but I think this code will flow better by keeping the indentation consistent.